CHANGELOG.md

```diff
@@ -64,6 +64,14 @@ This was added in [pull request #2230: Add extra letter spacing modifier for inp
 
 ### Deprecated features
 
+#### Stop using JavaScript API methods other than the `init` method
+
+All JavaScript methods in the API for our JavaScript components except `init` are now deprecated, and will be made private in v5.0.
+
+If you're calling a method other than `init`, [please tell us by filling in this form](https://docs.google.com/forms/d/e/1FAIpQLSfmH2AitMeouXqB0FWC5p5e6y1TSiFCjmJ8DrVuwfmpRGCaWw/viewform?usp=sf_link). We'll use this information when prioritising future additions to the public API.
```

Comment:
Please review the content and structure of the linked Google Form as well. I've pinged @claireashworth to get her thoughts on it too.